query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/QueryBuilder.java

```diff
@@ -400,7 +400,18 @@ public static Literal literal(@Nullable Object value) {
    */
   @NonNull
   public static Literal literal(@Nullable Object value, @NonNull CodecRegistry codecRegistry) {
-    return literal(value, (value == null) ? null : codecRegistry.codecFor(value));
+    try {
+      return literal(value, (value == null) ? null : codecRegistry.codecFor(value));
```

Comment:
is `literal` throwing CodecNotFoundException if the codec passed to it as an `argument == null`?
Maybe we can handle that at this higher level without the need to throw an exception?

Reply:
It throws if `codecFor` doesn't find a codec to handle `value`.

If `value` is null, `codecFor` is not called so CNFE is never thrown.